libs/ui-components/src/components/AuthProvider/CreateAuthProvider/utils.ts

```
@@ -0,0 +1,548 @@
+      .of(Yup.string())
+      .test('unique-scopes', t('Please remove duplicate scopes'), (scopes) => {
+        const uniqueScopes = new Set(scopes);
+        return uniqueScopes.size === scopes?.length;
+      }),
+    usernameClaim: Yup.array().of(Yup.string()),
+    roleAssignmentType: Yup.string().oneOf(Object.values(RoleAssignmentType)),
+    roleClaimPath: Yup.array()
+      .of(Yup.string())
+      .when('roleAssignmentType', {
+        is: RoleAssignmentType.Dynamic,
+        then: (schema) => schema.min(1, t('At least one claim path segment is required for dynamic role assignment')),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)